plugins/node.d/icinga_hosts

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf 
+
+if [ "$1" = "autoconf" ]; then
+    if [ -r /usr/bin/icingacli ]; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli)"
```

Comment:
I would recommend to use something like the following in the header of the script:
```shell
ICINGACLI=${ICINGACLI:-$(which icingacli)}

Later you can check, whether "$ICINGACLI" is empty or the referenced exists.
This allows to use the plugin in different environments (with different paths) simply by supplying an adjusted default configuration (without changing the code of the plugin).
```